src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -383,8 +394,8 @@ export const ArtistAuctionResultsRefetchContainer = createRefetchContainer(
         aggregations={
           props.aggregations as SharedAuctionResultsFilterContextProps["aggregations"]
         }
-        earliestCreatedYear={startAt}
-        latestCreatedYear={endAt}
+        earliestCreatedYear={startAt ? parseInt(startAt) : null}
```

Comment:
is it possible to return a NaN from the parse?

Reply:
Nope we are not aggregating null values [see here](https://github.com/artsy/diffusion/blob/main/app/queries/lot_filtered_query.rb#L51)